File: poultry_pal_plus_app/lib/models/egg_packaging_record.dart
class EggPackagingRecord {
  final String id;
  final String createdDate;
  final String eggSize;
  final String boxSize;
  final int numberOfBoxes;
  final int totalEggs;
  final String userId;
  final String additionalInfo;

  EggPackagingRecord({
    required this.id,
    required this.createdDate,
    required this.eggSize,
    required this.boxSize,
    required this.numberOfBoxes,
    required this.totalEggs,
    required this.userId,
    required this.additionalInfo,
  });

  factory EggPackagingRecord.fromJson(Map<String, dynamic> json) {
    return EggPackagingRecord(
      id: json['id'],
      createdDate: json['createdDate'],
      eggSize: json['eggSize'],
      boxSize: json['boxSize'],
      numberOfBoxes: json['numberOfBoxes'],
      totalEggs: json['totalEggs'],
      userId: json['userId'],
      additionalInfo: json['additionalInfo'] ?? '',
    );
  }
}

File: poultry-pal-service/src/main/java/co/za/hlaluko/dynamics/poultry/pal/model/dto/EggPackagingRecord.java
package co.za.hlaluko.dynamics.poultry.pal.model.dto;

import java.util.Date;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class EggPackagingRecord {
  private String id;
  private Date createdDate;
  private String eggSize;
  private String boxSize;
  private int numberOfBoxes;
  private int totalEggs;
  private String userId;
  private String additionalInfo;
}
